docs/content/ingestion/firehose.md

```diff
@@ -38,7 +38,58 @@ See [Examples](../tutorials/examples.html). This firehose creates a stream of ra
 #### RabbitMqFirehose
 
 This firehose ingests events from a define rabbit-mq queue.
+<br>
+**Note:** Add **amqp-client-3.2.1.jar** to lib directory of druid to use this firehose.
```

Comment:
this library is part of the extension dependency, If we set the classpath properly before using the methods this should work, I believe its more like a classpath issue than the docs.